services/configuration/src/transactions.rs

```diff
@@ -321,3 +388,25 @@ impl Transaction for Vote {
         Ok(())
     }
 }
+
+impl Transaction for VoteAgainst {
+    fn verify(&self) -> bool {
+        self.verify_signature(self.from())
+    }
+
+    fn execute(&self, fork: &mut Fork) -> ExecutionResult {
+        let vote_against = VotingDecisionRef::from(self);
+        let _parsed_config = vote_against.precheck(fork.as_ref()).map_err(|err| {
```

Comment:
Nitpick: you can just remove `let _parsed_config = ` part.

Reply:
Done